connector/protobuf/src/test/resources/protobuf/functions_suite.proto

```diff
@@ -170,4 +170,41 @@ message timeStampMsg {
 message durationMsg {
   string key = 1;
   Duration duration = 2;
-}
\ No newline at end of file
+}
+
+message OneOfEvent {
+  string key = 1;
+  oneof payload {
```

Comment:
How do one-of fields look like in spark schema? Could you give an example? I could not see the schema in the unit tests.

Reply:
@rangadi the "Oneof" field is of message type, Oneof will be converted to a struct type.